core/cas-server-core-util-api/src/main/java/org/apereo/cas/util/CollectionUtils.java

```diff
@@ -97,6 +98,44 @@
         return c;
     }
 
+    /**
+     * Convert the object given into a {@link List} instead.
+     *
+     * @param obj the object to convert into a collection
+     * @return The collection instance containing the object provided
+     */
+    public static List<Object> toList(final Object obj) {
```

Comment:
This seems to be duplicating the toCollection method.

Reply:
very similar... it's needed to rebuild a `Set` into a `List` and cast the elements. I will rework the interface in another PR to get rid of this. Currently, only `public List<String> getEligibleAccountsForSurrogateToProxy(final String username)` uses this function. A Set will do just fine but requires redoing the interface.